<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEvent</class>
 <widget class="QDialog" name="DialogEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>事件表</string>
  </property>
  <widget class="QPushButton" name="butfind">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>查   询</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>721</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>22</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>22</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local5">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>29</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local1">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>29</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>23</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>22</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="wefw">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>22</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>22</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local4">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>29</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>21</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local2">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>28</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local6">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>29</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local3">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>29</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>27</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>从</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>27</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>到</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>24</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local2_2">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>29</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local6_2">
   <property name="geometry">
    <rect>
     <x>706</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="wefw_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>23</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>23</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>738</x>
     <y>23</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>22</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local5_2">
   <property name="geometry">
    <rect>
     <x>657</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local4_2">
   <property name="geometry">
    <rect>
     <x>607</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>23</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local3_2">
   <property name="geometry">
    <rect>
     <x>558</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local1_2">
   <property name="geometry">
    <rect>
     <x>457</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_44">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>23</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="display">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>346</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butclear">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>373</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空显示</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>382</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>381</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>日志文件名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butsave">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>374</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>存为
文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
